src/shared/services/localstorage.ts

```diff
@@ -18,13 +18,15 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { dehydrate } from 'services/duckUtils'
```

Comment:
is this not needed here now?

Reply:
The matching hydrate code that adds a flag on redux states is never called in current version, so it should be safe to never call dehydrate to remove the flag. As far as I can see, it is safe to remove the hydrate/dehydrate functions completely, they seem to only be used in their test cases. But I left them in this PR to not have a large diff to review.

Reply:
👍 I wasnt sure what it does really.